controllers/propagator/propagation.go

```diff
@@ -408,7 +432,5 @@ func (r *ReplicatedPolicyReconciler) processTemplates(
 		}
 	}
 
-	log.V(1).Info("Successfully processed templates")
```

Comment:
This log message is removed because `processTemplates` is now always called and when no templates are present, it does nothing. This is to clean up watches if hub templates used to be present but are now removed.